src/todo/Lists.js

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect } from "react";
+import { connect } from "react-redux";
+import * as actions from "../store/actions/list";
+import * as addTodoInListActions from '../store/actions/add_todo_in_list';
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Paper,
+  makeStyles,
+  Container,
+  Typography
+} from "@material-ui/core";
+
+import AddIcon from "@material-ui/icons/Add";
+import EditIcon from "@material-ui/icons/Edit";
+import DeleteForeverIcon from "@material-ui/icons/DeleteForever";
+
+const useStyles = makeStyles({
+  table: {
+    minWidth: 600,
+  },
+});
+
+const Lists = (props) => {
+  const classes = useStyles();
+  const handleEditList = (val) => {
+    props.handleEditList(val);
+  };
+  console.log("Lists is rendered");
+  console.log(props);
+  const handleDeleteList = (id) => {
+    props.deleteList(id);
+  };
+
+  const handleToggleAndId=(id)=>{
+    props.isVisibleAddTodoInList(true);
+    props.setIdForAddingToDo(id);
+  }
+  useEffect(() => {
+    props.allLists();
+    console.log("UseEffect is called in list to dos");
+  }, []);
```

Comment:
Here you have used props object again and again.

You can destructure the props
https://medium.com/@lcriswell/destructuring-props-in-react-b1c295005ce0

const Lists = ( { handleEditList,deleteList,isVisibleAddTodoInList ...   } ) => {

Reply:
yes sir i know but i am using here, done.